org.kde.kate.json

```diff
@@ -0,0 +1,46 @@
+{
+    "id": "org.kde.kate",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "kate",
+    "rename-icon": "kate",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=cups",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=host"
```

Comment:
is there an 'open by folder' or project type of functionality that needs this? or could it all theoretically work with the existing portal?

Reply:
Yes, it has a project functionality